plugins/main/bridge/bridge.go

```diff
@@ -379,6 +388,25 @@ func setupVeth(netns ns.NetNS, br *netlink.Bridge, ifName string, mtu int, hairp
 	return hostIface, contIface, nil
 }
 
+func removeDefaultVlan(hostVeth netlink.Link) error {
+	vlanInfo, err := netlink.BridgeVlanList()
+	if err != nil {
+		return err
+	}
+
+	brVlanInfo, ok := vlanInfo[int32(hostVeth.Attrs().Index)]
+	if ok {
+		for _, info := range brVlanInfo {
+			err = netlink.BridgeVlanDel(hostVeth, info.Vid, false, true, false, true)
```

Comment:
question @mlguerrero12: do any of these boolean attributes even matter for deleting the VLAN ?

Reply:
The pvid and untagged flags are ignored

   bridge vlan delete - delete a vlan filter entry
       This command removes an existing vlan filter entry.

       The arguments are the same as with bridge vlan add.  The pvid and
       untagged flags are ignored.